ai_agents/agents/ten_packages/extension/soniox_asr_python/extension.py

```
@@ -89,6 +92,12 @@ async def on_init(self, ten_env: AsyncTenEnv) -> None:
                 category=LOG_CATEGORY_KEY_POINT,
             )
 
+            if self.config.finalize_mode == FinalizeMode.IGNORE:
+                assert (
+                    "enable_endpoint_detection" in self.config.params
+                    and self.config.params["enable_endpoint_detection"]
```

Ruff (0.14.0)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)